sebs/gcp/datastore.py

```
@@ -0,0 +1,180 @@
+                        "gcloud firestore databases create "
+                        f" --database='{database_name}' "
+                        f" --location={self.region} "
+                        f" --type='datastore-mode' "
+                    )
+                    self.logging.info(f"Allocated a new Firestore database {database_name}")
+
+                else:
+
+                    self.logging.error("Couldn't query Datastore instances!")
+                    self.logging.error(e)
+                    raise RuntimeError("Couldn't query Datastore instances!")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)